addons/sys_core/fnc_remoteStartSpeaking.sqf

```diff
@@ -82,7 +82,27 @@ private _result = false;
     };
 
     _unit setVariable [QGVAR(ts3id), _speakingId];
+
+    if (_languageId in (call FUNC(getSpokenLanguages))) then {
+        private _previousLanguage = _unit getVariable [QGVAR(languageId), -1];
+        if (_previousLanguage == -1) then {
+            if (ACRE_CURRENT_LANGUAGE_ID != _previousLanguage) then {
```

Comment:
Isn't this redundant? Can ACRE_CURRENT_LANGUAGE_ID  be - 1?

Reply:
On line 87 it uses -1 as the default value, this will indicate that this is the first time this person has been encountered with voice

Reply:
Oh nevermind I see what you mean, yes that it out of place